lib/glueby/contract/payment.rb

```diff
@@ -18,7 +18,7 @@ module Contract
     # => 0(tapyrus)
     #
     # Send
-    # Payment.transfer(sender: sender, receiver: receiver, amount: 10_000)
+    # Payment.transfer(sender: sender, receiver_address: receiver.internal_wallet.receive_address, amount: 10_000)
```

Comment:
You are free to derive the address in any way you like, so here is a sample that specifies a string instead of `receiver.internal_wallet.receive_addres`.